configs/gpt4all.yaml

```diff
@@ -1,13 +1,11 @@
 llm:
   provider: gpt4all
-  model: 'orca-mini-3b.ggmlv3.q4_0.bin'
   config:
+    model: 'orca-mini-3b.ggmlv3.q4_0.bin'
     temperature: 0.5
     max_tokens: 1000
     top_p: 1
     stream: false
 
 embedder:
-  provider: gpt4all
-  config:
-    model: 'all-MiniLM-L6-v2'
```

Comment:
Any specific reason to remove from the yaml file? We want to keep it here so that users can refer to on how to configure the embedding model.

Reply:
So langchain's GPT4ALL embedder does not configure with model name, they create embeddings for all the models they support. No need to pass the model name to configure the embedder.

Reply:
Wait, I dont know if I understand. Isn't creating embeddings for all the models is expensive?

Reply:
Behind the scene they user `gpt4all` packages `Embed4All` class to create embeddings for models. Langchain is just wrapping it around its own class `GPT4AllEmbeddings`. Need to further investigate to learn how they can create embeddings with the correct dimensions that works for different models that they offer.